components/Feed/index.js

```diff
@@ -61,15 +92,63 @@ const greetingSubscription = gql`
   }
 `
 
+const dateFormat = timeFormat('%A,\n%d.%m.%Y')
+
+const groupByDate = nest().key(d => dateFormat(new Date(d.meta.publishDate)))
+
 class Feed extends Component {
+  constructor (props) {
+    super(props)
+    this.nodes = []
```

Comment:
I think this shouldn't be an instance variable. If you wanted to have access to the filtered nodes array: move the filter to the `props` fn of the `graphql` wrapper:
```js
      props: ({ data }) => ({
        data,
        nodes: data.documents && data.documents.nodes.filter(), // is safe because `DocumentConnection.nodes` defined as a mandatory array (`[]!`)
```